File: HTTPTest2/src/main/java/server/Executable.java
package server;

public class Executable {
    private String code;
    private String unitTests;
    private boolean executeTests;

    public String getCode() {
        return code;
    }

    public String getUnitTests() {
        return unitTests;
    }

    public boolean getExecuteTests() {
        return executeTests;
    }

    enum ExecutionType {CODE, UNIT_TESTS};

    public Executable(String code, String unitTests, boolean executeTests) {
        this.code = code;
        this.unitTests = unitTests;
        this.executeTests = executeTests;
    }
}

File: Code/ExecutionEnvironments/CSharp/NetCoreExecutionEnvironment/Contracts/Executable.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace NetCoreExecutionEnvironment.Contracts
{
    public class Executable
    {
        public string code { get; set; }
        public string unitTests { get; set; }
        public bool executeTests { get; set; }
    }
}
